Makefile.in

```diff
@@ -56,7 +56,7 @@ ifneq ($(XLEN),32)
 GITVER := $(shell git --git-dir=$(GCC_SRCDIR)/.git describe --tag --always)
 PKGVER := "ARCv3 ARC64 GNU/Linux $(GITVER)"
 make_tuple = arc64-unknown-$(1)
-LINUX_TUPLE  ?= $(call make_tuple,linux-gnu)
+LINUX_TUPLE  ?= arc64-linux-gnu
```

Comment:
I guess that historically we created "arc-snps-linux-" and then created "arc-linux-" soft links.
Should we change this ?

Reply:
a tuple is something like `cpu-company-os-clibrary`. Maybe the best is just to remove the `unknown` from above.

Reply:
@cupertinomiranda indeed the ideal way is to do what you suggest, but the existing toolchain build scripts don't create any symlinks hence my fix to just remove "unknown". Note that more robust buildsystems (e.g. Buildroot) already do the right thing for arc64.